tests/e2e/forms/repeat-form.wdio-spec.js

```diff
@@ -133,6 +134,34 @@ describe('RepeatForm', () => {
     }
   });
 
+  describe('Repeat form with select', () => {
+    it('should display the initial form and its repeated content in the default language', async () => {
+      const swUserName = 'Jina la mtumizi';
+      await loginPage.changeLanguage('sw', swUserName);
+      await login();
+      await openRepeatForm(selectFormDocument.internalId);
+
+      const washingtonInputPath = '#report-form input[name="/cascading_select/selected_state"][value="washington"]';
+      const washingtonLabelPath = `${washingtonInputPath} ~ .option-label.active`;
+      expect(await (await $(washingtonLabelPath)).getText()).to.equal('Washington');
+
+      await (await $(washingtonInputPath)).click();
+      const kingInputPath = '#report-form input[name="/cascading_select/selected_county"][value="king"]';
+      const kingLabelPath = `${kingInputPath} ~ .option-label.active`;
+      expect(await (await $(kingLabelPath)).getText()).to.equal('King');
+
+      await (await $(kingInputPath)).click();
+      const seattleLabelPath = getCityLabelPath('seattle');
+      const redmondLabelPath = getCityLabelPath('redmond');
+      expect(await (await $(seattleLabelPath)).getText()).to.equal('Seattle');
+      expect(await (await $(redmondLabelPath)).getText()).to.equal('Redmond');
+    });
+
+    function getCityLabelPath(cityValue) {
+      return `#report-form input[name="/cascading_select/selected_city"][value="${cityValue}"] ~ .option-label.active`;
```

Comment:
you can consider expanding this one to receive the field name - `selected_city` and make it return `{ input, label }`
The you can use it for all these fields (L144, L145, L149, L150, L154 and L155) :)